server/services/prisonerService.ts

```diff
@@ -127,8 +125,12 @@ export default class PrisonerService {
     )
   }
 
-  async hdcLicenceIsApproved(bookingId: string): Promise<boolean> {
-    const hdcLicence = await this.getActiveHdcStatus(bookingId)
+  async isHdcApproved(bookingId: number): Promise<boolean> {
+    const prisoners = await this.prisonerSearchApiClient.searchPrisonersByBookingIds({ bookingIds: [bookingId] })
+    if (!prisoners.length || !prisoners[0].homeDetentionCurfewActualDate) {
+      return false
```

Comment:
they can no longer be classed for HDC if they are no longer eligible, regardless what NOMIS says.